simple_history/models.py

```diff
@@ -467,6 +484,8 @@ def get_meta_options(self, model):
         meta_fields["verbose_name"] = name
         if self.app:
             meta_fields["app_label"] = self.app
+        if self._date_indexing == "composite":
+            meta_fields["index_together"] = (("history_date", model._meta.pk.attname),)
```

Comment:
I believe this change would necessitate a DB change via a migration. If this is enabled, does Django complain about model changes that don't have migrations created for it?

Reply:
See below, it does not.